gems/activerecord/7.0/_test/test.rbs

```diff
@@ -3,5 +3,6 @@ module Test
   end
 
   class User < ActiveRecord::Base
+    extend ActiveRecord::Encryption::EncryptableRecord::ClassMethods
```

Comment:
You may want to `extend` the `ActiveRecord::Base` class, not the user defined record class?